git_pull_request/bitbucket.py

```diff
@@ -0,0 +1,196 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import daiquiri
+import requests
+
+
+LOG = daiquiri.getLogger("git-pull-request")
```

Comment:
```suggestion
LOG = daiquiri.getLogger(__name__)
```
Maybe? Not sure if there's a downside here.